test/core/gen2/local/test_usercode.py

```diff
@@ -104,7 +104,7 @@ def test_class_bad_params(self):
             "\n"
             "Failed validating 'minimum'"
             " in schema['properties']['value']:\n"
-            "    {'minimum': 1, 'type': 'integer'}\n"
+            "    {'type': 'integer', 'minimum': 1}\n"
```

Comment:
This test did not fail on my local machine, but did fail in the CI. Adjusting it, fixed the CI. Still I do not understand this behavior.

Reply:
I guess, what is formatted here is not an ordered dict, so the order of fields may change from build to build or even run to run.